edn/src/types.rs

```diff
@@ -83,8 +83,8 @@ impl Display for Value {
             }
             Map(ref v) => {
                 write!(f, "{{")?;
-                for (key, val) in v {
-                    write!(f, " :{} {}", key, val)?;
+                for (index, (key, val)) in v.iter().enumerate() {
```

Comment:
The leading colon is definitely wrong, so good fix there, but generally EDN does not include commas -- they're whitespace.  Why would we include commas in `Map` but not the other container types?  Because of the pairing?  I'd prefer not to include them, which agrees with Clojure{Script?, although I could be argued down.  @rnewman, do you have a strong opinion?